src/main/java/org/jenkinsci/backend/ircbot/IrcBotConfig.java

```diff
@@ -14,37 +14,37 @@
  * @author Oleg Nenashev <o.v.nenashev@gmail.com>
  */
 public class IrcBotConfig {
-    
+
```

Comment:
IMHO removing whitespace isn't refactoring by definition. But I also believe in leaving source code cleaner than you found it. If you do not want this commit I can remove it.